rotkehlchen/inquirer.py

```diff
@@ -654,8 +652,8 @@ def _find_usd_price(
             underlying_tokens = asset.underlying_tokens
 
             # Check if it is a special token
-            if asset.identifier in instance.special_tokens:
-                ethereum = instance.get_evm_manager(chain_id=ChainID.ETHEREUM)
+            if asset.identifier in Inquirer.special_tokens:
+                ethereum = Inquirer().get_evm_manager(chain_id=ChainID.ETHEREUM)
```

Comment:
why do you need the parentheses in this one?

Reply:
Because is a method and not an staticmethod. I've changed it